bindings/wasm/crate/src/lib.rs

```diff
@@ -131,6 +131,47 @@ pub fn decrypt_message(
     }
 }
 
+// This function is the most barebones (incorrect) implementation of obtaining
+// a "contact bundle" for a given VoodooIdentity by handle. The output string
+// is a JSON object which can subsequently be imported into another VoodooInstance
+#[wasm_bindgen]
+pub fn get_public_account_json(handle: &str) -> Result<String, JsValue> {
+    let instances = INSTANCE_MAP.lock().unwrap();
```

Comment:
Instead of `unwrap` you can use `?`. lock returns a LockResult which should convert to result.

Reply:
yup!